src/scene/text-bitmap/asset/loadBitmapFont.ts

```diff
@@ -96,7 +96,8 @@ export const loadBitmapFont = {
 
             textureUrls.push({
                 src: imagePath,
-                data: textureOptions
+                data: textureOptions,
+                parser: imagePath.startsWith('blob:') ? 'texture' : undefined
```

Comment:
It would be much more useful if users could declare the parser to use rather than assuming every blob is a texture. 

```ts
Assets.load({ src: "path/to/font.fnt", data: { imageParser: "ktx" } })
```

This way it would easily support ktx, basis and other compressed texture types.